cashews/serialize.py

```diff
@@ -109,6 +109,17 @@ async def set(self, key: str, value: Any, *args: Any, **kwargs: Any):
         value = self._pickler.dumps(value)
         return await super().set(key, self._prepend_sign_to_value(key, value), *args, **kwargs)
 
+    async def set_many(self, pairs: Dict[str, Any], *args: Any, **kwargs: Any):
```

Comment:
the main Interface using `data` name instead of `pairs`  ( but I would prefer to use `pairs` everywhere)